extensions-core/google-extensions/src/test/java/org/apache/druid/storage/google/GoogleStorageTest.java

```diff
@@ -62,31 +65,39 @@ public void setUp()
   }
 
   @Test
-  public void testDeleteSuccess() throws IOException
+  public void testDeleteSuccess()
   {
     EasyMock.expect(mockStorage.delete(EasyMock.eq(BUCKET), EasyMock.eq(PATH))).andReturn(true);
     EasyMock.replay(mockStorage);
     googleStorage.delete(BUCKET, PATH);
   }
 
   @Test
-  public void testDeleteFailure()
+  public void testDeleteFileNotFound()
   {
     EasyMock.expect(mockStorage.delete(EasyMock.eq(BUCKET), EasyMock.eq(PATH))).andReturn(false);
     EasyMock.replay(mockStorage);
-    boolean thrownIOException = false;
+    googleStorage.delete(BUCKET, PATH);
+  }
+
+  @Test
+  public void testDeleteFailure()
+  {
+    EasyMock.expect(mockStorage.delete(EasyMock.eq(BUCKET), EasyMock.eq(PATH))).andThrow(STORAGE_EXCEPTION);
+    EasyMock.replay(mockStorage);
+    boolean thrownStorageException = false;
```

Comment:
Same, we can replace the try-catch + boolean logic with this one liner:
```suggestion
    assertThrows(StorageException.class, () -> googleStorage.delete(BUCKET, PATH));
```

Reply:
Done.